docs/cni/kubernetes/VagrantCoreOS.md

```diff
@@ -0,0 +1,149 @@
+<!--- master only -->
+> ![warning](../images/warning.png) This document applies to the HEAD of the calico-containers source tree.
+>
+> View the calico-containers documentation for the latest release [here](https://github.com/projectcalico/calico-containers/blob/v0.13.0/README.md).
+<!--- else
+> You are viewing the calico-containers documentation for release **release**.
+<!--- end of master only -->
+
+# Deploying Calico and Kubernetes on CoreOS using Vagrant and VirtualBox
+
+These instructions allow you to set up a Kubernetes v1.1.4 cluster with [Calico networking][calico-networking] using Vagrant and the [Calico CNI plugin][calico-cni]. This guide does not setup TLS between Kubernetes components.
+
+## 1. Deploy cluster using Vagrant 
+
+### 1.1 Install dependencies
+
+* [VirtualBox][virtualbox] 5.0.0 or greater.
+* [Vagrant][vagrant] 1.7.4 or greater.
+* [Git][git]
+
+<!--- master only -->
+### 1.2 Clone this project
+
+    git clone https://github.com/projectcalico/calico-containers.git
+<!--- else
+### 1.2 Clone this project, and checkout the **release** release
+
+    git clone https://github.com/projectcalico/calico-containers.git
+    git checkout tags/**release**
+<!--- end of master only -->
+    
+### 1.3 Startup and SSH
+
+Edit `calico-containers/docs/cni/cloud-config/node-config.yaml` and uncomment the line in `/etc/hosts` so that the nodes can resolve the hostname `kubernetes-master`.  The line should look like this:
```

Comment:
What should it resolve to?  I get that I need to change the IP, but change it to the IP of what?  The host that I'm running vagrant on?